src/main/webapp/index.html

```diff
@@ -48,6 +48,13 @@ <h1>About us:</h1>
                 <li>The purpose of this project was not only to provide a website that can prove to be useful, but also was an opportunity to enhance and strengthen our programming skills.</li>
             </ul>
 
+            <div>
+                <h1>Top Recipes:</h1>
+                <button onclick="addRandomRecipe()"</button>
```

Comment:
Consider using `<body onload` property instead, so the recipe is automatically displayed on the homepage without any user intervention: https://www.w3schools.com/tags/ev_onload.asp